editor/plugins/script_text_editor.cpp

```diff
@@ -544,7 +544,7 @@ void ScriptTextEditor::_validate_script() {
 
 		if (errors.size() > 0) {
 			// TRANSLATORS: Script error pointing to a line and column number.
```

Comment:
This comment seems no longer necessary.

Reply:
I'd argue the more TRANSLATORS comments the merrier but, yeah.